reminders/class_models/notification_models.py

```diff
@@ -0,0 +1,63 @@
+from events.models import EventParticipant
+from django.utils import timezone
+from django.db import models
+from django.db.models import Q
+from django.db.models.functions import Now
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
```

Comment:
I think you can get rid of line 7 and import the IntegrityError in line 3, like this:
```suggestion
from django.db import models, IntegrityError
from django.db.models import Q
from django.db.models.functions import Now
from django.core.exceptions import ValidationError
```

Reply:
done. 😄